client/locales/de.json

```diff
@@ -0,0 +1,402 @@
+{
+    "common": {
+        "ok": "OK",
+        "submit": "Übernehmen",
+        "cancel": "Abbrechen",
+        "overwrite": "Überschreiben",
+        "continue": "Weiter",
+        "dismiss": "Abbrechen",
+        "prompt": "Hinweis",
+        "warning": "Warnung",
+        "attention": "Achtung!",
+        "confirm": "Bestätigen",
+        "close": "Schließen",
+        "save": "Speichern",
+        "cancelled": "Abgebrochen",
+        "enabled": "Eingeschaltet",
+        "commandOK": "Befehl erfolgreich übertragen!",
+        "commandFailed": "Es ist ein Fehler aufgetreten:",
+        "connectionError": "Verbindungsfehler",
+        "m2": "m²",
+        "hours": "Stunde",
+        "hours_plural": "Stunden",
+        "months": ["Jan","Feb","Mär","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dez"],
+        "days": ["Montag","Dienstag","Mittwoch","Donnerstag","Freitag","Samstag","Sonntag"]
+    },
+    "navi": {
+        "home": "Home",
+        "map": "Karte",
+        "zones": "Zonen",
+        "control": "Steuerung",
+        "settings": "Einstellungen",
+        "links": "Links",
+        "about": "Über"
+    },
+    "home": {
+        "labels": {
+            "area": "Bereich",
+            "time": "Zeit",
+            "battery": "Batterie",
+            "state": "Ladungszustand..."
+        },
+        "buttons": {
+            "start": "Start",
+            "pause": "Pause",
+            "stop": "Stop",
+            "home": "Home",
+            "spot": "Spot",
+            "find": "Finde",
+            "goto": "Gehe zu",
+            "zones": "Zonen",
+            "storeMap": "Karte speichern",
+            "loadMap": "Karte laden",
+            "unknownPower": "Unbekannte Leistung",
+            "invalid": "Ungültiger Button"
+        },
+        "popups": {
+            "selectFanspeed": "Auswahl des Leistungsmodus:",
+            "selectGotoTarget": "Setzte einen Zielpunkt:",
+            "selectZones": "Auswahl der Zonen zur Reinigung:",
+            "selectMapToLoad": "Auswahl des Speicherplatzes zum Laden der Karte:",
+            "emptyZoneChosen": "Es wurde eine leere Zone gewählt. Bitte zuerst die Zone einrichten.",
+            "tooManyZones": "Es wurde versucht {{count}} Unterzonen zu wählen. Es werden maximal 5 unterstützt.",
+            "chooseMapName": "Titel zum Speichern der Karte wählen:",
+            "invalidMapName": "Der Titel darf nur Buchstaben, Zahlen, Leerzeichen und Bindestriche enthalten.",
+            "slotAlreadyTaken": "Speicherplatz #{{slot}} ist bereits in Benutzung",
+            "nameAlreadyTaken": "Name \"{{name}}\" wurde schon verwendet",
+            "noMapsStored": "Keine gespeicherte Karten!",
+            "slotNo": "Speicherplatz #",
+            "confirmMapDelete": "Soll \"{{name}}\" wirklich gelöscht werden?",
```

Comment:
Maybe "Karte \"{{name}}\" wirklich löschen?", this would include the reference that you delete a map for the user - in case someone uses the same name for a map and another item.